lib/at_cmd_parser/at_cmd_parser.c

```diff
@@ -343,3 +329,29 @@ int at_parser_max_params_from_str(const char *at_params_str,
 
 	return err;
 }
+
+enum at_cmd_type at_parser_cmd_type_get(const char *at_cmd)
+{
+	enum at_cmd_type type;
+
+	if (!is_command(at_cmd)) {
+		return AT_CMD_TYPE_UNKNOWN;
+	}
+
+	at_cmd += sizeof("AT+") - 1;
```

Comment:
There is one AT command, "AT". Will this be OK?

Reply:
Right, I guess we might need some special logic to cover it, I'll have a look.

Reply:
Move two bytes, then call is_lfcr() to confirm the termination of AT string. If not move one further.

A similar problem in static inline bool is_command(const char *str).